src/rusthello/main.rs~

```diff
@@ -0,0 +1,118 @@
+//! A simple Reversi game written in Rust with love
```

Comment:
Sorry, apparently git and I don't like each other… Should I do anything to fix these?